src/app/components/audio-analysis/pages/list/list.component.ts

```diff
@@ -1,23 +1,119 @@
 import { Component } from "@angular/core";
+import { ActivatedRoute } from "@angular/router";
+import { AnalysisJobsService } from "@baw-api/analysis/analysis-jobs.service";
+import { Filters } from "@baw-api/baw-api.service";
+import { projectResolvers } from "@baw-api/project/projects.service";
 import {
   audioAnalysesMenuItem,
   audioAnalysisCategory,
-  newAudioAnalysisMenuItem,
+  newAudioAnalysisJobMenuItem,
 } from "@components/audio-analysis/audio-analysis.menus";
-import { PageComponent } from "@helpers/page/pageComponent";
+import { PagedTableTemplate } from "@helpers/tableTemplate/pagedTableTemplate";
+import { Param } from "@interfaces/apiInterfaces";
+import { AnalysisJob } from "@models/AnalysisJob";
+import { Project } from "@models/Project";
 import { List } from "immutable";
+import { DateTime } from "luxon";
+import { Observable } from "rxjs";
+
+interface TableRow {
+  name: Param;
+  scripts: AnalysisJob;
+  creator: AnalysisJob;
+  started: DateTime;
+  status: string;
+  statusUpdated: DateTime;
+  model: AnalysisJob;
+}
+
+const projectKey = "project";
 
-//TODO: OLD-CLIENT REMOVE
 @Component({
   selector: "baw-audio-analyses",
-  template: "<baw-client></baw-client>",
+  templateUrl: "list.component.html",
 })
-class AudioAnalysesComponent extends PageComponent {}
+class AudioAnalysesComponent extends PagedTableTemplate<TableRow, AnalysisJob> {
+  public constructor(
+    protected api: AnalysisJobsService,
+    protected route: ActivatedRoute
+  ) {
+    super(api, (analysisJobs) =>
+      analysisJobs.map((analysisJob) => ({
+        name: analysisJob.name,
+        scripts: analysisJob,
+        creator: analysisJob,
+        started: analysisJob.startedAt,
+        status: analysisJob.overallStatus,
+        statusUpdated: analysisJob.overallStatusModifiedAt,
+        model: analysisJob,
+      }))
+    );
+  }
+
+  public columns = [
+    { name: "Id" },
+    { name: "Name" },
+    { name: "Scripts" },
+    { name: "Creator" },
+    { name: "Started" },
+    { name: "Status" },
+    { name: "Status Updated" },
+    { name: "Actions" },
+  ];
+
+  public sortKeys = {
+    id: "id",
+    name: "name",
+    started: "startedAt",
+    status: "overallStatus",
+    statusUpdated: "overallStatusModifiedAt",
+  };
+
+  public get project(): Project | null {
+    const routeProjectModel = this.route.snapshot.data[projectKey];
+    if (routeProjectModel) {
+      return routeProjectModel.model;
+    }
+
+    return null;
```

Comment:
why would it ever by null?

didn't I see on the admin component you overrode the project property?

Reply:
Yeah, it should never be null, but this is just defensive programming

If I didn't return `null` here, the `Project` model would be typed as `private project: Project | null | undefined;` (because no return is implicit `undefined`.

In hind sight, it might be good to throw an error here so we hard fail if the project (somehow) fails to initialize.